template/INSTRUCTIONS.md.tpl

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+This document lays out the next steps for AWS service teams upon bootstrapping the ACK {{ .ServicePackageName }}-controller repository successfully.
+
+1. To get started, look inside the `generator.yaml` file, ignore the resource(s) from the resource list and execute `make build-controller` from the ACK code-generator.
```

Comment:
I don't understand what this step is asking the user to do. `Ignore the resource` sounds like it's saying don't take any action? And shouldn't the CLI have already run `make build-controller`?

Reply:
This has been modified to: “1. To get started, look inside the `generator.yaml` file, and remove the resource(s) from the ignore list and execute `make build-controller` from the ACK code-generator.”